test_util/marathon.py

```diff
@@ -9,7 +9,17 @@
 
 import test_util.helpers
 
+DEFAULT_API_BASE = '/marathon/v2'
```

Comment:
a whole bunch of pods stuff is going to want slightly different stuff from /marathon/v2 I think (Although should confirm with that team).